pkg/exporter/scheduler_metrics.go

```diff
@@ -32,6 +32,7 @@ type SchedulerMetrics struct {
 	SpinningThreads      prometheus.Gauge
 	IdleThreads          prometheus.Gauge
 	GlobalRunQueueLength prometheus.Gauge
+	NeedsSpinningThreads prometheus.Gauge
```

Comment:
```suggestion
	NeedSpinningThreads prometheus.Gauge
```
Let's change this name and subsequent metric names as well because the `schedtrace` output is as follows: `needspinning`

Reply:
Done